.github/workflows/manual-deploy-testnet-l2.yml

```diff
@@ -297,6 +298,22 @@ jobs:
       - update-loadbalancer
       - deploy-l2-contracts
     steps:
-      - name: 'Trigger Faucet deployment workflow'
+      - name: 'Trigger Faucet deployment on workflow dispatch'
+        if: ${{ github.event_name == 'workflow_dispatch' }}
         run: |
           curl -XPOST -H "Authorization: Bearer ${{ secrets.GH_TOKEN }}" -H "Accept:application/vnd.github" -H "Content-Type:application/json" https://api.github.com/repos/obscuronet/faucet/actions/workflows/manual-deploy-${{ github.event.inputs.testnet_type }}-faucet.yml/dispatches --data '{"ref": "main" }'
+
+      - name: 'Trigger Faucet deployment on schedule'
+        if: ${{ github.event_name == 'schedule' }}
+        run: |
+          curl -XPOST -H "Authorization: Bearer ${{ secrets.GH_TOKEN }}" -H "Accept:application/vnd.github" -H "Content-Type:application/json" https://api.github.com/repos/obscuronet/faucet/actions/workflows/manual-deploy-dev-testnet-faucet.yml/dispatches --data '{"ref": "main" }'
+
+  run-e2e-tests:
+    runs-on: ubuntu-latest
+    needs:
+      - deploy-faucet
+    steps:
+      - name: 'Run E2E tests on schedule trigger after successful deployment'
+        if: ${{ success() && github.event_name == 'schedule' }}
```

Comment:
What does the `success()` mean here ?

Reply:
Just means if the run is successful up to this point ...

Reply:
... to be fair it's implied so probably can be removed.